test/suite/test_txn25.py

```diff
@@ -34,7 +34,7 @@
 from wtscenario import make_scenarios
 
 class test_txn25(wttest.WiredTigerTestCase):
-    conn_config = 'cache_size=50MB,log=(enabled)'
+    conn_config = 'cache_size=50MB'
 
```

Comment:
I think for this one it might be good to run it both with and without logging. It's about write generations (has no timestamps) and it's reasonable to suppose that the behavior might be affected by logging.

Separately, without logging it probably ought to checkpoint before aborting the second transaction (and maybe evict some pages too) or likely no transaction IDs will be written out at all anyway.

Reply:
Fixed in 1600ba2ad and b01c938b7.

Can you please review again, this is a relatively larger change and there might be a simpler way to configure both with and without logging that what I used?

I added a checkpoint in all cases to force writing with a transaction ID, but there are only 1000 k/v pairs in the test, I'm not clear on whether forced eviction of  a page or two is going to be useful.